src/App.tsx

```diff
@@ -10,19 +10,15 @@ import Navigation from "./components/pages/Navigation";
 import NameForm from "./components/pages/StatedNameForm";
 import GraphForm from "./components/pages/StatedGraphForm";
 import ChartFunction from "./components/ChartFunctional";
-import { toJS } from "mobx";
 
 
 
-const App = observer(({ balanceService }: {balanceService: BalancesServices}) => {
+const App = observer(({ balanceService }: {balanceService: BalanceService}) => {
 
   return (
     <div className="App">
           <StatedBalanceForm balanceService={BalanceService.instance}/>
-          {/* <Chart
-              values={toJS(balanceService)}
-              /> */}
-          <ChartFunction values={toJS(balanceService)} />
+          <ChartFunction values={balanceService.getBalances()} />
```

Comment:
call to getBalances() will be moved to StatedChart, when we create one.
 
Btw. so you understand what happens here:
When App component uses any data from balanceService, and data changes, then mobX rerenders it.
MobX sees usage of balanceSerbice, because its child (ChartFunction) accesses balanceService.balances values (which are passed by getBalances).
I'm not sure however if it is a good approach because I think that the whole App gets rerendered, not only ChartFunction. <we need to check it>
So in the future we will make only StatedChart to be observer and App will be not.